packages/astro/src/core/routing/astro-designed-error-pages.ts

```diff
@@ -15,6 +15,7 @@ export const DEFAULT_404_ROUTE: RouteData = {
 	route: '/404',
 	fallbackRoutes: [],
 	isIndex: false,
+	origin: 'core',
```

Comment:
Although we use "core" to refer to the `astro` package, is this as clear to users of the API as it is to us? Maybe `origin: 'astro'` would be better for users.